components/private_channel/BUILD.gn

```diff
@@ -0,0 +1,32 @@
+import("//brave/components/private_channel/buildflags.gni")
+import("//build/buildflag_header.gni")
+
+buildflag_header("buildflags") {
+  header = "buildflags.h"
+  flags = [ "PRIVATE_CHANNEL_ENABLED=$private_channel_enabled" ]
+}
+
+source_set("private_channel") {
+  public_deps = [ ":buildflags" ]
+
+  sources = [
+    "browser/constants.cc",
```

Comment:
if this component is browser process only then these files should be at the top level - https://www.chromium.org/developers/design-documents/cookbook

Reply:
- [ ] refactor `constant.*` and `utils.*` to be at the top level